fuzz/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+# spell-checker:ignore libfuzzer
+
+[package]
+name = "uucore-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+
+[dependencies.uucore]
+path = "../src/uucore/"
+
+[dependencies.uu_date]
+path = "../src/uu/date/"
```

Comment:
Can we use the workspace dependency instead of path?
```
[dependencies]
uu_date = { workspace = true }
```
I guess it comes down to the same thing, though.

Reply:
@tertsdiepraam for, uu_date = { workspace = true }
will use the one from crates.io